scripts/run-registry.ts

```diff
@@ -92,7 +92,7 @@ const publish = async (packages: { name: string; location: string }[], url: stri
             const command = `cd ${path.resolve(
               '../code',
               location
-            )} && yarn pack --out=${PACKS_DIRECTORY}/${tarballFilename} && cd ${PACKS_DIRECTORY} && npm publish ./${tarballFilename} --registry ${url} --force --access restricted --ignore-scripts`;
+            )} && yarn pack --out=${PACKS_DIRECTORY}/${tarballFilename} && cd ${PACKS_DIRECTORY} && npm publish ./${tarballFilename} --registry ${url} --force --ignore-scripts`;
```

Comment:
For some reason, it is impossible to build sandboxes locally in no-link mode. It seems, that non-scoped packages cannot be released in a restricted way

Reply:
I ran into the same problem in the CPC branch, but I figured it was due to something I changed.

I resolved it the same way; there seems no reason to restrict access.